### PR TITLE
add note about copying folder

### PR DESCRIPTION
**Pull request by KarthikSiddarth**
Note on copying a directory from one branch to another

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1591,6 +1592,12 @@ $ rm -rf .git/modules/submodulename
 
 ## Miscellaneous Objects
 
+### Copy a folder from one branch to another
+
+```sh
+$ (my-branch)$ git checkout <branch-you-want-the-directory-from> --<folder-name>
```
Comment (RichardLitt) [inline README.md]: You need a space between the dash and the carrot in  `--<folder-name>`. This also also works for a file from another branch; perhaps add a sentence clarifying that?

Comment (KarthikSiddarth) [reply README.md]: sure

**Comment by KarthikSiddarth** [comment]
made changes as per the request

**Comment by RichardLitt** [comment]
Cheers!

**Pull request merged**